python/format/pyproject.toml

```diff
@@ -0,0 +1,19 @@
+[project]
+name = "format"
```

Comment:
Should this be called sth like validator instead?

Reply:
Seen together: for the moment we keep the current name, because data verification and data generation are very entangled, and it's not clear at this stage how we will split them in two libraries.